java-checks/pom.xml

```diff
@@ -18,6 +18,11 @@
       <artifactId>sonar-plugin-api</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+    <groupId>org.springframework</groupId>
+      <artifactId>spring-expression</artifactId>
+      <version>6.1.1</version>
+    </dependency>
```

Comment:
We will want to move this dependency version in the dependency management of the top-level pom

Reply:
Fixed.